### DISCUSSION TITLE
[FR] Add 'New Row' UI element

### DISCUSSION THREAD

**Discussion by wsydney76** [craftcms/cms#11328] [Ideas] [answered]
Example: You have a field layout like this:

Row 1: Lightswitch Field A (width 1/4), Field B (1/4, Condition: Field A is true), Field C (1/2, Condition: Field A is true)
Row 2: Field X (1/2) Field Y (1/2)

If field A is false, it will change to:

Row 1: Lightswitch Field A (width 1/4),  Field X (1/2) 
Row 2: Field Y (1/2)

which doesn't make sense from a UX perspective.

Would be nice to have a 'New Row' UI element that forces field X to stay in row 2, but without having a heavy separator like a heading or horizontal rule.

/Mel

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 4.1 🎉 

<img width="268" alt="A tab within a field layout designer, with two 25% width fields, followed by a “Line Break” UI element, followed by a third field with 25% width." src="https://user-images.githubusercontent.com/47792/170728920-0f03dee6-2eea-493f-8b0f-b905ba276faa.png">

**Reply by brandonkelly** [reply]
Craft 4.1 is out with this!
